bootstrap/terraform/redis.tf

```diff
@@ -0,0 +1,20 @@
+resource "null_resource" "setup_redis_docker" {
+  triggers = {
+    always = timestamp()
+  }
+
+  provisioner "local-exec" {
+    command = "cd ../../scripts && ./gen-test-certs.sh && docker-compose up -d"
+  }
+}
```

Comment:
We could add a resource to handle the make env-teardown command so the docker image gets cleaned-up:
```
resource "null_resource" "teardown_redis_docker" {
  triggers = {
    always = timestamp()
  }

  provisioner "local-exec" {
    when = destroy
    command = "docker stop vault_redis_test && docker rm vault_redis_test"
  }
}
```